bot/__main__.py

```diff
@@ -31,17 +32,18 @@
     )
 )
 
-bot.load_extension("bot.cogs.antivirus")
-bot.load_extension("bot.cogs.error_handler")
-bot.load_extension("bot.cogs.filter")
-bot.load_extension("bot.cogs.information")
-bot.load_extension("bot.cogs.owner")
-bot.load_extension("bot.cogs.snekbox")
-bot.load_extension("bot.cogs.tags")
-
-bot.load_extension("bot.cogs.moderation.infractions")
-bot.load_extension("bot.cogs.moderation.member_log")
-bot.load_extension("bot.cogs.moderation.message_log")
-bot.load_extension("bot.cogs.moderation.silence")
+cogs = [
+    f[:-3] for f in listdir("bot/cogs") 
+    if f.endswith(".py")
+]
+moderation = [
+    f[:-3] for f in listdir("bot/cogs/moderation")
+    if f.endswith(".py")
+]
+
+for loadfiles in cogs:
```

Comment:
I recommend you to rename `loadfiles` to `cog`.

Reply:
ah ok